kyo-scheduler/jvm/src/main/scala/kyo/scheduler/util/Top.scala

```diff
@@ -24,16 +24,22 @@ class Top(
     private val mBeanServer: MBeanServer = ManagementFactory.getPlatformMBeanServer
     private val objectName               = new ObjectName("kyo.scheduler:type=Top")
 
-    if (enableTopJMX) {
-        close()
-        mBeanServer.registerMBean(new StandardMBean(this, classOf[TopMBean]), objectName)
-    }
+    private var lastConsoleStatus: Scheduler.Status = null
 
     if (enableTopConsoleMs > 0)
         timer.schedule(enableTopConsoleMs.millis) {
-            println(Top.print(status()))
+            val currentStatus = status()
```

Comment:
I assume `status()` will never return `null`?

If not, should we have a `Status.empty` to prevent gaps in reporting? That would prevent the need for null on line 27 as well.

Reply:
I'm not sure there's much value in reporting the initial measurement since it'd present accumulated counters up to the point when the observation started.

Reply:
Right, wasn't sure if status could also be null for any other reason.